packages/ldc/build.sh

```diff
@@ -38,7 +37,7 @@ termux_step_post_extract_package () {
 	mv tools-${TERMUX_PKG_VERSION[2]} rdmd
 	mv dub-${TERMUX_PKG_VERSION[3]} dub
 
-	export LLVM_TRIPLE=$TERMUX_HOST_PLATFORM
+	export LLVM_TRIPLE=${TERMUX_HOST_PLATFORM/-/--}
```

Comment:
This was needed because of a strange issue where LLVM didn't like that the default triple Termux provides didn't have a vendor listed, so I made it a quadruple with the vendor blank. This used to work fine before LLVM 7, seems like a regression. I don't know why it doesn't affect clang.